test/KubeOps.Operator.Test/Events/EventPublisher.Integration.Test.cs

```diff
@@ -40,7 +40,7 @@ public async Task Should_Create_New_Event()
         e.Metadata.Annotations.Should().Contain(a => a.Key == "originalName" && a.Value == eventName);
     }
 
-    [Fact]
+    [Fact(Skip = "Does not work for some reason")]

```

Comment:
Let's try online :-)

My setup (with windows and macos) is to use docker desktop kubernetes extension. However, it should work with Kind and minikube as well.